Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -382,8 +382,18 @@ theorem IsNormal.le_iff {f} (H : IsNormal f) {a b} : f a ≤ f b ↔ a ≤ b :=
 theorem IsNormal.inj {f} (H : IsNormal f) {a b} : f a = f b ↔ a = b := by
   simp only [le_antisymm_iff, H.le_iff]
 
+theorem IsNormal.le_apply {f} (H : IsNormal f) (a) : a ≤ f a :=
```

Comment:
```suggestion
theorem IsNormal.le_apply {f} (H : IsNormal f) {a} : a ≤ f a :=
```
since one can't rewrite with this lemma and `a` is never specified in applications

Reply:
I'm confused. Isn't the general rule to make any argument that can't be inferred from the others explicit?

Reply:
Yes, but there are many exceptions to this rule, eg iffs and inequalities. Pragmatically, no use of `le_apply` every needed to specify `a`, and anyone wanting to specify it could use `id_le` instead. Also note that the new `(_ := _)` notation means explicitly specifying implicit arguments is much less of a pain than it used to be